identity/did/identity.go

```diff
@@ -129,8 +132,28 @@ func logTxHash(tx *types.Transaction) {
 	log.Infof("Transfer pending: 0x%x\n", tx.Hash())
 }
 
-func (i identity) AddKey(key Key) (chan *ethereum.WatchTransaction, error) {
-	contract, opts, err := i.prepareTransaction(*i.did)
+func (i identity) getDID(ctx context.Context) (did DID, err error) {
+	tc, err := contextutil.Account(ctx)
+	if err != nil {
+		return did, err
+	}
+
+	addressByte, err := tc.GetIdentityID()
+	if err != nil {
+		return did, err
+	}
+	did = NewDID(common.BytesToAddress(addressByte))
```

Comment:
Do we need to expose `NewDID` ?

Reply:
I have to think about how exactly this will replace CentID.

Reply:
It should be multi-step
1. lets make CentID type to `common.Address` and rename it to `identity.DID`
2. Change all references of `did.DID` to `identity.DID` in did package and wherever used